pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -525,16 +562,8 @@ func buildHTTPConnectionManager(opts buildListenerOpts) *http_conn.HttpConnectio
 		Name: xdsutil.Router,
 	})
 
-	/*	TODO(mostrowski): need to port internal build functions for mixer.
-		if opts.mesh.MixerCheckServer != "" || opts.mesh.MixerReportServer != "" {
-			mixerConfig := v1.BuildHTTPMixerFilterConfig(opts.mesh, opts.proxy, opts.proxyInstances, opts.outboundListener, opts.store)
-		filter := &http_conn.HttpFilter{
-			Name: v1.MixerFilter,
-			Config:MessageToStruct(mixerConfig),
-		}
-			filters = append([]*http_conn.HttpFilter{filter}, filters...)
-		}
-	*/
+	filters = append(filters, opts.hTTPFilters...)
```

Comment:
fyi, this is incorrect. router filter should be the last filter in the chain.

Reply:
done